docs/application/dotnet/guides/nui/text.md

```diff
@@ -24,15 +24,15 @@ The `TextLabel` class is lightweight, non-editable, and do not respond to user i
 
 To create a text label:
 
-1.  Create an instance of the [Tizen.NUI.BaseComponents.TextLabel](https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.TextLabel.html) class and define the label text as a parameter:
+1.  Create an instance of the `TextLabel` class and define the label text as a parameter. To make text visible, set `TextColor` property to `Color.White`:
 
     ```csharp
-    Window window = Window.Instance;
-    TextLabel label = new TextLabel("Hello World");
-    window.Add(label);
+        TextLabel label = new TextLabel("Hello World");
+        label.TextColor = Color.White;
+        Window.Instance.Add(label);

```

Comment:
Please match with starting point of \`.

4 spaces in front of each line should be removed.

![image](https://user-images.githubusercontent.com/9958374/97862375-8e77d180-1d48-11eb-84ff-3354e4ab487d.png)